packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -19,6 +19,7 @@ import {
   fileURLToFilePath,
 } from '../utils/path.js';
 import { encodeInput, hasStatusCode } from './utils.js';
+import path from 'path';
```

Comment:
```diff
- import path from 'path';
+ import path from 'node:path';
```
But, this file should be independent from Node.js.
That's why we made `utils/path.ts`.